element_array_ephys/readers/spikeglx.py

```diff
@@ -14,6 +14,7 @@
 IMAX = {
     "neuropixels 1.0 - 3A": 512,
     "neuropixels 1.0 - 3B": 512,
+    "NP1015": 512,
```

Comment:
This will fix only this one case, what about other probe types, let's see if we can come up with a more robust and future proof solution?